### PR TITLE
#[158355400] Component to display errors in the application

### PR DESCRIPTION
**Pull request by Chris-Maina**
#### What does this PR do?
Creates a component to be used to display error messages.
#### How should this be manually tested?
`git clone` or `git pull` this repo or branch, `cd` into the project folder, run `yarn install` followed by `yarn start` to start the application. Login and navigate to `Redemptions`, if there's an error, you'll receive an error message with try again option.  
#### What are the relevant pivotal tracker stories?
[158355400](https://www.pivotaltracker.com/story/show/158355400)
#### Screenshots (if appropriate)
screen shot 2018-06-14 at 22 15 56

### CODE REVIEW

**Comment by Chris-Maina** [comment]
@kimani-ndegwa

File: src/common/ErrorMessage.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ErrorMessage = (props) => {
+  const { message, retry, param } = props;
+  return (
+    <div className='error-message'>
+      <img src='https://goo.gl/1VpMZd' alt='error-icon' />
```
Comment (kimani-ndegwa) [inline src/common/ErrorMessage.jsx]: Can we have such images as constants outside the component and then import them where necessary? Especially moving forward.

Comment (Chris-Maina) [reply src/common/ErrorMessage.jsx]: Okay, that can be done.

**Comment by kimani-ndegwa** [comment]
@Chris-Maina Please update this branch

**Comment by Chris-Maina** [comment]
@kimani-ndegwa On it.

**Pull request merged**